Packs/PrismaCloud/ReleaseNotes/1_9_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Prisma Cloud (RedLock)
+- changed the url to https://api.prismacloud.io/
+- changed fetch incidents to support pagination
+- fixed bug in fetch incidents with last fetch time
```

Comment:
bug is a forbidden word in rn. use only issue.
```suggestion
- Changed the default *Server URL* to: https://api.prismacloud.io/
- Added pagination to the fetch incidents.
- Fixed an issue where the fetch failed in case there was a large number of non fetched alerts.```